src/App.jsx

```diff
@@ -1,11 +1,53 @@
 import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import DonationForm from './DonationForm';
 
 const App = () => {
-  return (
-    <>
-      <p>Hello World</p>
-    </>
-  );
+  const { backend } = useBackend();
+  var max_id = -1;
+
+  const [formData, setFormData] = useState({
+    business_id: null, // Since there's no business_id in the form and the database requires a business id (foreign key), the default for now is the maximum business id in the database (9)
+    canned_cat_food_quantity: null,
+    canned_dog_food_quantity: null,
+    date: null,
+    donation_id: null,
+    dry_cat_food_quantity: null,
+    dry_dog_food_quantity: null,
+    email: null,
+    food_bank_donation: null,
+    misc_items: null,
+    reporter: null,
```

Comment:
im unable to highlight everything here but move these functions and api related stuff into the DonationForm.jsx. This is because no other file will use this api related content so we dont need it lingering around in app.jsx like this. There exists an alternative option where you can create a DonationFormApi.js file but its just a matter of preference, and our whole team is keeping their API within the jsx file so for consistenty sake lets do that.